src/components/comment-input.component.js

```diff
@@ -113,12 +120,12 @@ export class CommentInput extends Component {
                   : translate('issue.main.commentInput', language)
               }
               multiline
-              blurOnSubmit
+              blurOnSubmit={false}
               onChangeText={text => this.setState({ text })}
               onContentSizeChange={event =>
                 this.setState({ height: event.nativeEvent.contentSize.height })}
               onSubmitEditing={event =>
```

Comment:
But the `onSubmitEditing` handler is no more required if we have `blurOnSubmit={false}`